packages/bits/src/lib/menu/menu-key-control.service.ts

```diff
@@ -146,12 +148,15 @@ export class MenuKeyControlService implements OnDestroy {
     }
 
     private handleClosedKeyDown(event: KeyboardEvent): void {
+        event.stopPropagation();
+
         // prevent opening on enter and prevent scrolling page on key down/key up when focused
         if (this.shouldBePrevented(event)) {
             event.preventDefault();
         }
 
         if (event.code === KEYBOARD_CODE.ARROW_DOWN) {
+            event.preventDefault();
```

Comment:
I think this call of `preventDefault()` is redundant because we call it in the first `if` statement